### ISSUE TITLE
macos big sur network issues is incompatible w/ lulu 2.3.1

### ISSUE THREAD

**Issue by a1exus** [objective-see/LuLu#360]
Hello,

I updated my macOS to macOS Big Sur 11.3 and LuLu to 2.3.1 and now I'm getting following (through Terminal app)

```
alexus@mbp ~ % time host google.com
;; connection timed out; no servers could be reached
host google.com  0.00s user 0.00s system 0% cpu 10.010 total
alexus@mbp ~ % grep ^nameserver /etc/resolv.conf 
nameserver 192.168.86.1
alexus@mbp ~ % netstat -rn | egrep 'default.*en0'
default            192.168.86.1       UGScg          en0       
alexus@mbp ~ % time ping -c1 192.168.86.1
PING 192.168.86.1 (192.168.86.1): 56 data bytes

--- 192.168.86.1 ping statistics ---
1 packets transmitted, 0 packets received, 100.0% packet loss
ping -c1 192.168.86.1  0.00s user 0.00s system 0% cpu 11.014 total
alexus@mbp ~ % 
```

as soon as I disable, everything works fine.

---

REF: https://unix.stackexchange.com/questions/647365/macos-big-sur-network-issues

**Comment by jdswinbank** [comment]
See also #355.

**Comment by darkolo** [comment]
Same issue with 11.4

**Comment by objective-see** [comment]
This was likely due a bug in macOS. 
Others have reported that this (or at least other similar issues) were addressed by Apple in macOS 11.5. 

Can you upgrade and see if that addresses this issue? Thanks!

**Comment by a1exus** [comment]
Lulu 2.3.1 seems to be working with macOS 11.5
